Formula/licensefinder.rb

```diff
@@ -15,9 +16,7 @@ class Licensefinder < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "8fad2104fa2186844235e5628d8afe03f34b08e7b9c62631399dde2b3947b493"
   end
 
-  on_system :linux, macos: :mojave_or_older do
-    depends_on "ruby@2.7"
-  end
+  uses_from_macos "ruby", since: :catalina
```

Comment:
Ruby 3 support was added in 7.0.0 - https://github.com/pivotal/LicenseFinder/commit/02497dfb60d458e51a43ef26568389b5fcf302af